interpreter/cling/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
       "--bindir"
       "--libdir"
       "--includedir"
-      "--prefix")
+      "--prefix"
+      "--shared-mode")
```

Comment:
Doesn't this means that now Cling is always build in shared mode?  Should this be made optional?

Reply:
No, that asks the llvm config how it was built.

Reply:
Perfect.